CHANGELOG.rst

```diff
@@ -43,6 +43,8 @@ This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
 * Add validation for project information input for offer creation/update (`#488 <https://github.com/qbicsoftware/offer-manager-2-portlet/issues/488>`_)
 
+* Use ``life.qbic.business.logging.Logging`` instead of console print outs (`#537 <https://github.com/qbicsoftware/offer-manager-2-portlet/issues/537>`_)
+
```

Comment:
I think this is not something that would need to be part of the CL 🤔

Reply:
You might be right